app/styles/_projects.less

```diff
@@ -0,0 +1,66 @@
+//
+// Projects page
+// --------------------------------------------------
+
+.project-bar {
+  .project-sort {
+    .form-group {
+      margin: 0 0 0 0;
+    }
+  }
+
+  .projectSearch {
+    flex: 1;
+
+    @media (min-width: @screen-sm-min) {
+      flex: 0;
+      min-width: 200px;
+    }
+  }
+
+  .vertical-divider {
```

Comment:
You can probably drop this first rule that is only doing the `display:none` and add a [boostrap util class](http://getbootstrap.com/css/#responsive-utilities-classes) `.hidden-sm` to the element.